src/debugger/direct/directDebugSession.ts

```diff
@@ -46,6 +46,31 @@ export class DirectDebugSession extends DebugSessionBase {
             .then(() => {
                 logger.log("Launching the application");
                 logger.verbose(`Launching the application: ${JSON.stringify(launchArgs, null , 2)}`);
+<<<<<<< HEAD
```

Comment:
It seems that there is an unresolved conflict.

Reply:
Thanks for pointing. Will fix.